documentation/src/main/markdown/components/detectors.dita

```diff
@@ -824,6 +824,38 @@
 							</entry>
 							<entry>HIGH</entry>
 						</row>
+						<row>
+							<entry>
+								<b>Setuptools</b>
+							</entry>
+							<entry> </entry>
+							<entry> </entry>
+							<entry> </entry>
+							<entry> </entry>
+							<entry> </entry>
+						</row>
+						<row>
+							<entry> </entry>
+							<entry>Setuptools Detector</entry>
+							<entry>Python</entry>
+							<entry>PyPI</entry>
+							<entry>
+								<p>Files: pyproject.toml and optionally setup.cfg, or setup.py</p>
+								<p>Executables: python and pip, or python3 and pip3 specified via --detect.python.path and --detect.pip.path properties.</p>
```

Comment:
We do not require python, and therefore don't need the detect.python.path property, at least at runtime. It's best practice to setup a virtual environment before running Detect, which would likely need Python but Detect doesn't actually use it.

Reply:
Ah, ok, I assumed incorrectly that Detect would need Python for pip. 
Will update accordingly.